.eslintignore

```diff
@@ -0,0 +1,3 @@
+react/src/js/components/Consonant/**/*.{spec.js,test.js}
```

Comment:
Need to create separate PR to fix all linting issues in unit tests as this has been excluded before from build.